src/utilities/spacing/spacing.scss

```diff
@@ -41,11 +47,17 @@
     }
   }
 
+  /**
+    * 1. Ideally we could use the margin-inline shorthand, but it is not as well
```

Comment:
Should probably just be `*-inline` shorthand (since this covers `margin` _and_ `padding`).

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/662#discussion_r416941077